thorlcr/slave/thslavemain.cpp

```diff
@@ -296,6 +296,15 @@ void setSlaveAffinity(unsigned processOnNode)
 
 int main( int argc, char *argv[]  )
 {
+    for (unsigned i=0;i<(unsigned)argc;i++) {
+        if (strcmp("--daemon",argv[i])==0 || strcmp("-d",argv[i])==0) {
+            if (daemon(1,0) || write_pidfile(argv[++i])) {
+                perror("Failed to daemonize");
+                return EXIT_FAILURE;
+            }
+            break;
+        }
+    }
```

Comment:
Does the ssh (frunssh) from init_thor work ok to start all remote slaves and know they are ok if they now daemonize themselves immediately ?